local-dev/api-data-watcher-pusher/api-watch-push.sh

```diff
@@ -42,13 +41,7 @@ watch_apidatafolder() {
         then
             echo "******* Found changes in gql files in /api-data/, clearing and re-populating"
 
-            if
-                send_graphql_query $clear_gql_file_path;
-            then
-                chsum_clear_prev=$chsum_clear_curr
-            else
-                echo '**** ERROR while clearing, will try again.'
-            fi
+            clear_data=$(/home/clear_api_data.py)
```

Comment:
I didn't see any messages from this script while testing so I think this method consumes any error messages from the python script, so maybe follow with a `echo $clear_data`. Or maybe there's a way to call the script differently that's allowed to print to stdout?

Also it doesn't look like the keycloak clearing script gets called anywhere? Would that go here?

Reply:
This should be resolved in 5ae039d239e68beec099fb8041c8712e7dfa8444